dynamodb/src/main/scala/zio/dynamodb/AttrMap.scala

```diff
@@ -1,5 +1,9 @@
 package zio.dynamodb
 
+import io.github.vigoo.zioaws.dynamodb.model.primitives.AttributeName
+import zio.dynamodb.DynamoDBExecutorImpl.awsAttributeValue
+import io.github.vigoo.zioaws.dynamodb.model.{ AttributeValue => ZIOAWSAttributeValue }
```

Comment:
`AttrMap` is a pure model ATM and this change will make it depend directly on `zio-aws` . I think we need to keep the separation of concerns between the model and the underlying DB client library as much as possible so that in future we could do things like swap out the DB lib more easily (eg for say a pure Scala lib if one becomes available) . Also the `AttrMap` model and machinery could potentially be extracted to its own module or lib in the future.

How about for now creating a function or an implicit extension method for AttrMap in DynamoDBExecutorImpl.scala where `toZioAwsMap` is used that takes an AttrMap and returns a `Map[AttributeName, ZIOAWSAttributeValue]` eg
```scala
  implicit class ToZioAwsMap(item: AttrMap) {
    def toZioAwsMap(): Map[AttributeName, ZIOAwsAttributeValue] =
      item.map.flatMap { case (k, v) => awsAttributeValue(v).map(a => (k, a)) }
  }
```
?

Reply:
This sounds a lot better to me, I'll make some changes tonight to clean it up.